codice/ddf
# DDF-5676 Remove Solr server from distribution

#### What does this PR do?
Removes included Solr server.
Adds SolrCloud docker compose file.
Updates itests to use single node SolrCloud.

#### Who is reviewing it? 

@lamhuy 
@leo-sakh

#### Select relevant component teams: 

@codice/solr 
@codice/docs 

#### Ask 2 committers to review/merge the PR and tag them here.

@ahoffer
@jrnorth
@ricklarsen

#### How should this be tested?

Follow instructions in `distribution/docker/solrcloud/README.md` to start SolrCloud.  Run DDF as normal and verify data can get indexed and searched.

Verify `Installing (Quick Start)` documentation for `Quick Install of SolrCloud` and `Quick Install of DDF` sections.

#### Any background context you want to provide?

#### What are the relevant tickets?
Fixes #5676 and #5640

#### Screenshots


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
